src/main.rs

```diff
@@ -100,6 +109,10 @@ fn main() -> anyhow::Result<()> {
         config,
     };
 
+    for seat in state.seats.iter().collect::<Vec<_>>() {
+        state.handle_seat_added(&mut conn, seat);
+    }
```

Comment:
This is not needed, as up until these lines of code the events have not been dispatched yet, so this loop will never iterate.

Reply:
I added that because of this note:
https://docs.rs/wayrs-utils/latest/wayrs_utils/seats/trait.SeatHandler.html#method.seat_added

> This is not called for seats that where advertised during the initial roundtrip.

Perhaps I misunderstood, though.

Reply:
Hmm, that comment looks outdated, thanks for pointing out.

Reply:
Done.